resources/ansible/roles/auditor/tasks/main.yml

```
@@ -2,7 +2,7 @@
 # This role assumes the existence of the node manager, which is setup using another role.
 - name: check if the sn_auditor binary exists
```

Should we adapt the comment above to match the check too?
```suggestion
- name: check if the auditor binary exists
```